packages/pangea-sdk/pangea/services/vault/vault.py

```diff
@@ -199,14 +206,51 @@ def secret_store(
             response.result = SecretStoreResult(**response.raw_result)
         return response
 
+    def pangea_token_store(
+        self,
+        pangea_token: str,
+        name: Optional[str] = None,
+        folder: Optional[str] = None,
+        metadata: Optional[Metadata] = None,
+        tags: Optional[Tags] = None,
+        rotation_policy: Optional[str] = None,
+        auto_rotate: Optional[bool] = None,
+        retain_previous_version: Optional[bool] = None,
```

Comment:
not needed for pangea tokens (the previous versions are in fact deleted from Pangea)